src/unix/pipe.c

```diff
@@ -285,9 +290,15 @@ int uv_pipe_connect2(uv_connect_t* req,
   memcpy(&saddr.sun_path, name, namelen);
   saddr.sun_family = AF_UNIX;
 
+  if (*name != '\0') {
+    addrlen = sizeof saddr;
+  } else {
+    addrlen = offsetof(struct sockaddr_un, sun_path) + namelen;
+  }
```

Comment:
Ditto, and no braces. Or alternatively:
```suggestion
    addrlen = sizeof saddr;
    if (*name == '\0')
      addrlen = offsetof(struct sockaddr_un, sun_path) + namelen;
```